shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/statement/dal/ChecksumTableStatement.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.sql.parser.sql.common.statement.dal;
+
+import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+
+/**
+ * Checksum table statement.
+ */
+public abstract class ChecksumTableStatement extends AbstractSQLStatement {
```

Comment:
Hi @strongduanmu Here `ChecksumTableStatement` is an abstract class. Does that mean we will have `MySQL ChecksumTableStatement`, or `OracleChecksumTableStatement`? But I only saw the definition of `MySQL ChecksumTableStatement`. Therefore, do you think the abstract class `ChecksumTableStatement` is required? Or maybe it is more suitable just to have a class named `MySQL ChecksumTableStatement`?

Other abstract classes have such an issue as well, so I would like to listen to what you think.

Reply:
@tristaZero This way is to ensure the consistency of the processing logic. Whether there is one subclass or multiple subclasses, it can be processed towards the parent class. Only when there is a special SQL dialect, the subclass will be used. 😂
Of course, when the SQL statement has only one dialect, this may cause users to misunderstand that it is a general statement.